dev/docker-compose.yaml

```diff
@@ -9,6 +9,8 @@ services:
       - be
       - be_next
       - oi
+    ports:
```

Comment:
do we ever need to expose this?

Reply:
Maybe not. I usually just run mongosh from within the pod rather than connecting from the host. I'll remove this.